src/main/java/de/hpi/bpt/argos/parsing/staticData/ArtifactParser.java

```diff
@@ -0,0 +1,26 @@
+package de.hpi.bpt.argos.parsing.staticData;
+
+import de.hpi.bpt.argos.parsing.XMLSubParser;
+
+/**
+ * This interface offers methods to parse artifacts from a XML file.
+ */
+public interface ArtifactParser<ArtifactType> extends XMLSubParser {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00119)